content/docs/guides/schema-diff.md

```diff
@@ -125,36 +125,4 @@ The `compare_schema` endpoint supports the following parameters:
 - `timestamp` / `base_timestamp` values must be provided in <LinkPreview href="https://en.wikipedia.org/wiki/ISO_8601" title="ISO 8601" preview="An international standard covering the worldwide exchange and communication of date and time-related data.">ISO 8601 format</LinkPreview>.
 </Admonition>
 
```

Comment:
I don't think the removal of content here is intended, and it leaves the the page in this state:

<img width="771" alt="image" src="https://github.com/user-attachments/assets/f3427381-6804-4785-b329-c95258ff552d" />